views/pulsar/v2/helpers/html.html.twig

```diff
@@ -308,7 +308,7 @@ data-*  | string | Data attributes, eg: `'data-foo': 'bar'`
                 })
             )
         }}>
-            {{- options.label|default|raw -}}
+            {{- options.label|default|striptags('<abbr><b><br><cite><code><dfn><em><i><img><kbd><q><small><span><strong><sub><sup>')|raw -}}
```

Comment:
I guess we manage this going forward for when the inevitable question comes in...

> How do I add a table as my label text

😄